### ISSUE TITLE
Wrong assumption: `this does not look like a fresh clone`

### ISSUE THREAD

**Issue by KES777** [newren/git-filter-repo#448]
```
this does not look like a fresh clone.
  (expected one remote, origin)
```

If there is only one remote, this does not mean that clone is not fresh.

**Comment by newren** [comment]
> If there is only one remote, this does not mean that clone is not fresh.

Yes, and clearly from the error message the program didn't think there was only one remote, so the rest of your statement isn't relevant.  Could you clarify how many remotes you have, from e.g. `git remote` output?

**Comment by KES777** [comment]
```
git remote -v
office	office-repo:office/bot (fetch)
office	office-repo:office/bot (push)
```
But I added only one remote:
```
$ git init
$ git remote add office office-repo:office/bot
$ git fetch
$ git checkout dev
$ git filter-repo ...
```

**Comment by newren** [comment]
> ```
> git remote -v
> office	office-repo:office/bot (fetch)
> office	office-repo:office/bot (push)
> ```
> 
> But I added only one remote:
> 
> ```
> $ git init
> $ git remote add office office-repo:office/bot
> $ git fetch
> $ git checkout dev
> $ git filter-repo ...
> ```
> 
> And here I get error about not fresh clone.

Ah, yeah, this is totally expected behavior.  See https://htmlpreview.github.io/?https://github.com/newren/git-filter-repo/blob/docs/html/git-filter-repo.html#FRESHCLONE  .  Note that the original statement, `this does not look like a fresh clone` was not `this is not a fresh clone`.

**Comment by KES777** [comment]
Nope, I do not expect this warning if repository is a fresh clone.
For example:
#1
```
git clone https://github.com/newren/git-filter-repo
cd git-filter-repo
git filter-repo --path=release/
```
No warning. Expected.

#2
```
git clone -o office https://github.com/newren/git-filter-repo
cd git-filter-repo
git filter-repo --path=release/
```
Warning. Not Expected. But repository is a fresh clone.

The better way to check backup copy is `fetch` before filtering and try to find remote label on first commit:
```
*   d4f43bc (HEAD -> main, office/main) XXXXXX
```
So we know that current state has a copy somewhere.

**Comment by KES777** [comment]
Another way to make sure a backup exists is to backup current branch:
```
git branch backup
git filter-repo
```

`filter-repo` could start new unrelated history and if something went wrong user always could switch back to `backup` branch.
